### PR TITLE
privateEndpointExampleEdits

### PR DESCRIPTION
**Pull request by Zuhairahmed**
## Description

Updated Example + Readme to make easier for new users to use example 

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Comment by Zuhairahmed** [comment]
@Adelmar92 got below error when tried to deploy this template, can you fix?

<html>
<body>


Resource handler returned message: "Error creating vcp Endpoint: InvalidVpcId.Malformed: Invalid Id: 'vpc-0a2e882f5b1103a14 \| atlasTest11302022' (expecting 'vpc-...; the Id may only contain lowercase alphanumeric characters and a single dash') status code: 400, request id: 48f44d65-6c8c-49fe-abcf-b24207c00dc7" (RequestToken: e39bc340-71c9-22e6-7ce3-2e84fb949dcd, HandlerErrorCode: GeneralServiceException)
--



</body>
</html>

**Comment by Adelmar92** [comment]
Hi zuhair? you where able to fix this? you sould only use the vpc-0a2e882f5b1103a14

> @Adelmar92 got below error when tried to deploy this template, can you fix?
> 
> ## Resource handler returned message: "Error creating vcp Endpoint: InvalidVpcId.Malformed: Invalid Id: 'vpc-0a2e882f5b1103a14 | atlasTest11302022' (expecting 'vpc-...; the Id may only contain lowercase alphanumeric characters and a single dash') status code: 400, request id: 48f44d65-6c8c-49fe-abcf-b24207c00dc7" (RequestToken: e39bc340-71c9-22e6-7ce3-2e84fb949dcd, HandlerErrorCode: GeneralServiceException)

File: examples/private-endpoint/privateEndpoint.json [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
+      },
+      "AWSRegion": {
+        "Type": "String"
+      },
+      "AWSVpcId": {
+        "Type": "String"
+      },
+      "AWSSubnetId": {
+        "Type": "String"
+      }
```
Comment (Adelmar92) [inline examples/private-endpoint/privateEndpoint.json:22]:
```suggestion
"MongoDBAtlasAPIPublicKey": {
        "Type": "String",
        "Description" : "MongoDB Atlas Public Access Key"
      },
      "MongoDBAtlasAPIPrivateKey": {
        "Type": "String",
        "Description" : "MongoDB Atlas Private Access Key"
      },
      "MongoDBAtlasProjectId": {
        "Type": "String",
        "Description" : "MongoDB project Key"
      },
      "AWSRegion": {
        "Type": "String",
        "Description" : "Cloud provider region for which you want to create the private endpoint service."
      },
      "AWSVpcId": {
        "Type": "String",
        "Default": "vpc-",
        "Description" : "AWS VPC ID (like: vpc-xxxxxxxxxxxxxxxx) (Used For Creating the AWS VPC Endpoint)"
      },
      "AWSSubnetId": {
        "Type": "String",
        "Default": "subnet-",
        "Description" : "AWS VPC Subnet ID (like: subnet-xxxxxxxxxxxxxxxxx) (Used For Creating the AWS VPC Endpoint)"
      }
```

**Review by Adelmar92 — APPROVED** [review]

**Comment by Zuhairahmed** [comment]
Thanks @Adelmar92!

**Pull request merged**